<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1430</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="RegWindowTab">
       <attribute name="title">
        <string>Registration</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_Register">
        <property name="geometry">
         <rect>
          <x>1080</x>
          <y>440</y>
          <width>311</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Perform Registration...</string>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider">
        <property name="geometry">
         <rect>
          <x>1370</x>
          <y>30</y>
          <width>22</width>
          <height>391</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>1351</width>
          <height>391</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QVTKWidget" name="QVTKRegFixed" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QVTKWidget" name="QVTKRegMoving1" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QVTKWidget" name="QVTKRegMoving2" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>1351</width>
          <height>32</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <item>
           <widget class="QPushButton" name="pushButton_SelectRegFixedDir">
            <property name="text">
             <string>Select Directory (Fixed) for Registration...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_SelectRegMoving1Dir">
            <property name="text">
             <string>Select Directory Time #1 (Moving) for Registration...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_SelectRegMoving2Dir">
         <property name="text">
          <string>Select Directory Time #2 (Moving) for Registration...</string>
         </property>
        </widget>
       </widget>
       <zorder>layoutWidget</zorder>
       <zorder>layoutWidget</zorder>
       <zorder>verticalSlider</zorder>
       <zorder>pushButton_Register</zorder>
       <zorder>pushButton_SelectRegMoving2Dir</zorder>
      </widget>
      <widget class="QWidget" name="SegWindowTab">
       <attribute name="title">
        <string>Segmentation</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_Segment">
        <property name="geometry">
         <rect>
          <x>1080</x>
          <y>440</y>
          <width>301</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Perform Segmentation...</string>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider_2">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>30</y>
          <width>22</width>
          <height>401</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>851</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_SelectSeg1Dir">
           <property name="text">
            <string>Select DICOM Directory #1 for Segmentation...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SelectSeg2Dir">
           <property name="text">
            <string>Select DICOM Directory #2 for Segmentation...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>851</width>
          <height>391</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QVTKWidget" name="QVTKSeg1" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QVTKWidget" name="QVTKSeg2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="CineViewerTab">
       <attribute name="title">
        <string>Cine Viewer</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_SelectCineDir">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>391</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Select DICOM Directory for Cine View ...</string>
        </property>
       </widget>
       <widget class="QVTKWidget" name="QVTKCineViewer" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>831</width>
          <height>491</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="MIPTab">
       <attribute name="title">
        <string>Maximum Intensity Projection</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_SelectMIPDir">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>391</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Select MIP DICOM Directory...</string>
        </property>
       </widget>
       <widget class="QVTKWidget" name="QVTKMIPViewer" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>831</width>
          <height>491</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1430</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindGrp">
    <property name="title">
     <string>WindGrp</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuWindGrp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
